### PR TITLE
Add missings translations + reorder some to simplify diff

### PR DESCRIPTION
**Pull request by Asenar**
Added the few missing translations and reordered a couple to simplify `diff Keyed-French.xml Keyed-English.xml`

### CODE REVIEW

**Comment by Asenar** [comment]
Sorry for the 2 commits, I compared files in root steam directory instead of 1.1. I noticed my mistake after, and I didn't know how to amend commit from windows.

File: Languages/French/Keyed/Keyed-French.xml [hunk 1]
```diff
@@ -25,9 +26,12 @@
     <Fluffy.ModManager.DependencyUnknownVersion>Dépend de {0}, mais sa version est inconnue.</Fluffy.ModManager.DependencyUnknownVersion>
     <Fluffy.ModManager.DependencyNotFound>Dépend de {0}, qui n'est pas chargé.</Fluffy.ModManager.DependencyNotFound>
     <Fluffy.ModManager.DependencyWrongVersion>Dépend de {0}, v{1} est chargé.</Fluffy.ModManager.DependencyWrongVersion>
```
Comment (b606) [inline Languages/French/Keyed/Keyed-French.xml]: English text here is `<Fluffy.ModManager.DependencyWrongVersion>Depends on {0} version {1}, version {2} is loaded.</Fluffy.ModManager.DependencyWrongVersion>`

File: Languages/French/Keyed/Keyed-French.xml [hunk 2]
```diff
@@ -25,9 +26,12 @@
...
+    <Fluffy.ModManager.DependencyNotActive>Depends de {0}, qui n'est pas chargé.</Fluffy.ModManager.DependencyNotActive>
     <Fluffy.ModManager.DependencyMet>{0} version {1} chargé.</Fluffy.ModManager.DependencyMet>
     <Fluffy.ModManager.IncompatibleMod>Incompatible avec {0}.</Fluffy.ModManager.IncompatibleMod>
+    <Fluffy.ModManager.LoadedBefore>Est chargé avant {0}.</Fluffy.ModManager.LoadedBefore>
     <Fluffy.ModManager.ShouldBeLoadedBefore>Doit être chargé avant {0}.</Fluffy.ModManager.ShouldBeLoadedBefore>
+    <Fluffy.ModManager.LoadedAfter>Est chargé après {0}.</Fluffy.ModManager.LoadedAfter>
```
Comment (b606) [inline Languages/French/Keyed/Keyed-French.xml]: You have extra tabs at the end of this line 34

**Comment by Asenar** [comment]
Thanks you @b606 for the review ! I only added the new translations but didn't checked the previous ones.

I usually works under linux, but I can't run RimWorld on it, so I only edit this file from windows (with github on notepad++) so I can test it.

**Comment by b606** [comment]
You are wecome !

NB. RimWorld runs fine on linux (at least on my debian stable + steam). However, some time ago, I had to run it from start_RimWorld_openglfix.sh or from the steam client.

**Comment by Karel-Kroeze** [comment]
thank you both very much! :)

**Pull request merged**